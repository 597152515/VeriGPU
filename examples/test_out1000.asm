# test writing to 1000 as stdout

addi x31, x0, 1000

li x1, 123
sw x1, 0(x31)

li x1, 44
sw x1, 0(x31)

li x1, 55
sw x1, 0(x31)

halt
