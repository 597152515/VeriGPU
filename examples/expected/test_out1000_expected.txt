out 0 00000000000000000000000001111011 0000007b 123
out 1 00000000000000000000000000101100 0000002c 44
out 2 00000000000000000000000000110111 00000037 55
